cmd/ddev/cmd/commands_test.go

```diff
@@ -347,6 +348,7 @@ func TestLaunchCommand(t *testing.T) {
 		"nested/path":                      app.GetPrimaryURL() + "/nested/path",
 		"/path-with-slash":                 app.GetPrimaryURL() + "/path-with-slash",
 		app.GetPrimaryURL() + "/full-path": app.GetPrimaryURL() + "/full-path",
+		":3000":                            primaryURLWithoutPort + ":3000",
```

Comment:
Should we add another test for ":3000/test"?

```suggestion
		":3000":                            primaryURLWithoutPort + ":3000",
		":3000/test":                     primaryURLWithoutPort + ":3000/test",
```

Reply:
I'm surprised it worked on gitpod, which is very picky about these things, and doesn't use `ddev-router`.

Reply:
As gitpod is special in its port handling, I'll be more concrete:

- `ddev launch`: opens https://8080-stasadev-ddev-ajl5zssk1i2.ws-eu110.gitpod.io (made sense to me)
- `ddev launch :3000`: opens https://3000-stasadev-ddev-ajl5zssk1i2.ws-eu110.gitpod.io, not found (made sense to me)
- `ddev launch :3000/test`: opens https://3000-stasadev-ddev-ajl5zssk1i2.ws-eu110.gitpod.io/test, not found (made sense to me)

Reply:
Also:

```console
gitpod /workspace/d10simple (main) $ DDEV_DEBUG=true ddev launch :3000
FULLURL http://127.0.0.1:3000
gitpod /workspace/d10simple (main) $ DDEV_DEBUG=true ddev launch :3000/test
FULLURL http://127.0.0.1:3000/test
```

Reply:
Thanks, I added more tests.